packages/react-jsx-highmaps/types/index.d.ts

```diff
@@ -0,0 +1,78 @@
+import type * as Highcharts from 'highcharts';
+import type { ReactElement, ReactNode } from 'react';
+import type {
+  HighchartsChartProps,
+  SeriesProps,
+  AxisProps
+} from 'react-jsx-highcharts';
+
+export * from 'react-jsx-highcharts';
```

Comment:
It think this should not export everything from react-jsx-highcharts as the real index.js only exports a subset:
https://github.com/whawker/react-jsx-highcharts/blob/918b92d60bd6cc4590095f7c79126e73581ae0a9/packages/react-jsx-highmaps/src/index.js#L2-L21

Reply:
I didn't see that bit in the index.js. I think the,
```js
export * from 'react-jsx-highcharts'; 
````
Should be
```js
 export { 
   Chart, 
   ColorAxis, 
   Credits, 
   Debug, 
   HighchartsProvider as HighmapsProvider, 
   Loading, 
   Legend, 
   Series, 
   Subtitle, 
   Title, 
   Tooltip, 
   useHighcharts, 
   useChart, 
   useAxis, 
   useSeries, 
   useModifiedProps, 
   withHighcharts as withHighmaps, 
   withSeriesType 
 } from 'react-jsx-highcharts'; 
```
instead. I can go and fix it so that the exports are more correct.

Reply:
I think that the useModifiedProps is exported accidentally in react-jsx-highmaps. It is for internal use only.